Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -109,6 +109,12 @@
     netsync: false
     sprite: Objects/Specific/Hydroponics/aloe.rsi
     state: cream
+  - type: Item
+    HeldPrefix: Aloe Cream
```

Comment:
this held prefix probably doesn't work, just add the stack stuff imo

Reply:
yeah wasn't sure about that aswell